### PR TITLE
add bet endpoints

### PR DESCRIPTION
**Pull request by rzeng95**
@nickweinberg

### CODE REVIEW

File: server/bets.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        SELECT *
+        FROM bets
+      `, (err = null, results) => {
+        return cb(err, JSON.parse(JSON.stringify(results)));
+      }
+    )
+  },
+
+  createBet: function(tournament_id, bettor_id, player_id, amount, cb) {
+    getConnection().query(`
```
Comment (DuaneGarber) [inline server/bets.js]: Update this to be like matches: https://github.com/nickweinberg/smash-dash/blob/master/server/matches.js#L4

Basically mysql/node is smart enough to take that object and convert it into an insertable row with magic

Comment (nickweinberg) [reply server/bets.js]: i dont like magic

**Comment by rzeng95** [comment]
@DuaneGarber fixed merge conflicts (docker setup works now)

**Comment by nickweinberg** [comment]
@rzeng95 YOLO NO TEST MERGE LETS GO!

**Pull request merged**